### PR TITLE
feat: adds other os install instruction for CLI

### PR DESCRIPTION
**Pull request by eaddingtonwhite**

### CODE REVIEW

File: docs/2-getting-started.md [hunk 1]
```diff
@@ -8,13 +8,27 @@ sidebar_position: 1.5
...
 ```
 brew tap momentohq/tap
 brew install momento-cli
 ```
 
+**Linux**
+
+1. Download the latest linux tar.gz file from [https://github.com/momentohq/momento-cli/releases/latest](https://github.com/momentohq/momento-cli/releases/latest)
+2. Unzip the file: `tar -xvf momento-cli-X.X.X.linux_x86_64.tar.gz`
+3. Move `./momento` to your execution path.
```
Comment (bruuuuuuuce) [inline docs/2-getting-started.md:23]: maybe provide basic linux command to do this?

Comment (bruuuuuuuce) [reply docs/2-getting-started.md:23]: can move to `bin` directory or add current directory to path

Comment (eaddingtonwhite) [reply docs/2-getting-started.md:23]: we were copying how stripe had for their cli docs. Was just keeping consistent with cli repo for now this was straight copy/pasta

Comment (bruuuuuuuce) [reply docs/2-getting-started.md:23]: :ack:

**Review by bruuuuuuuce — APPROVED** [review]
:shipit:

**Pull request merged**